nameserver/zone.go

```diff
@@ -18,6 +18,7 @@ var rdnsDomainLen = len(RDNSDomain) + 1
 
 type Zone interface {
 	AddRecord(ident string, name string, ip net.IP) error
+	DeleteRecords(ident *string, ip *net.IP, name *string) (count int)
```

Comment:
... and what about returning an `error` (in addition to the `count`)? All the other `Delete*` methods have it...